tests/js/components/__snapshots__/MetricsListItem.spec.jsx.snap

```diff
@@ -1,63 +0,0 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Unless I'm missing something, I think this sort of the purpose of snapshot testing (to capture what a rendered component looks like). Like say you accidentally changed a component such that it now renders differently. Unless you had something checked in that recorded how the component is supposed to look like, how would you know what it's supposed to look like?

From the [Jest documentation](https://facebook.github.io/jest/docs/snapshot-testing.html#snapshot-testing-with-jest): "The snapshot artifact should be committed alongside code changes, and reviewed as part of your code review process [...] On subsequent test runs Jest will simply compare the rendered output with the previous snapshot. If they match, the test will pass. If they don't match, either the test runner found a bug in your code that should be fixed, or the implementation has changed and the snapshot needs to be updated."